src/main/java/Du/Du.java

```diff
@@ -25,7 +28,7 @@ public Du(String filepath) {
 
 
     /**
-     * run Du itself
```

Comment:
Generally speaking, function descriptors should be more specific and more detailed, unless the function is a very generic one such as getters and setters. I would recommend adding in more details such as what purpose exactly do run do, perhaps 2 to 3 lines will do?